startup.sh

```diff
@@ -11,19 +15,27 @@ __     __        _  __         _   _       _        ____  ___
 EOF
 tput sgr0
 
-# Generate PKI and config if necessary
-if test ! -d data; then
-  command -v cfssl >/dev/null || brew install cfssl
-  ./generate/hub-dev-pki.sh
+# Build a docker image with all of our dependencies for future steps
```

Comment:
It might make more sense for this block to appear before the beautiful ascii art?

Reply:
Agreed